packages/mds-db/tests/mds-db.spec.ts

```diff
@@ -228,5 +230,50 @@ if (pg_info.database) {
         assert(!isNullOrUndefined(result.stats.current_running_queries))
       })
     })
+
+    describe('unit test policy functions', () => {
+      before(async () => {
+        const client: MDSPostgresClient = configureClient(pg_info)
+        await client.connect()
+        await dropTables(client)
+        await createTables(client)
+        await updateSchema(client)
+        await client.end()
+      })
+
+      after(async () => {
+        await MDSDBPostgres.shutdown()
+      })
+
+      it('can write, read, and publish a Policy', async () => {
+        await MDSDBPostgres.initialize()
+        await MDSDBPostgres.writePolicy(POLICY_JSON)
+        const result = await MDSDBPostgres.readPolicies({ policy_id: POLICY_JSON.policy_id })
+        assert.deepEqual(result[0], POLICY_JSON)
+
+        await MDSDBPostgres.writePolicy(POLICY2_JSON)
+        await MDSDBPostgres.publishPolicy(POLICY_JSON.policy_id)
+        const allPolicies = await MDSDBPostgres.readPolicies()
+        assert.deepEqual(allPolicies.length, 2)
+        const unpublishedPolicies = await MDSDBPostgres.readPolicies({ get_unpublished: true })
+        assert.deepEqual(unpublishedPolicies.length, 1)
+      })
+
+      it('can edit a Policy', async () => {
+        const policy = clone(POLICY2_JSON)
+        policy.name = 'a shiny new name'
+        await MDSDBPostgres.editPolicy(policy)
+        const result = await MDSDBPostgres.readPolicies({ policy_id: POLICY2_JSON.policy_id })
+        assert.deepEqual(result[0].name, 'a shiny new name')
+      })
+
+      it('will not edit a published Policy', async () => {
+        const publishedPolicy = clone(POLICY_JSON)
+        publishedPolicy.name = 'a shiny new name'
+        await MDSDBPostgres.editPolicy(publishedPolicy)
```

Comment:
`editPolicy` should probably throw an error if you try to edit a published policy, I think it'd be better to verify that an error is thrown instead of checking that the string for name hasn't changed.

Reply:
done